src/drem/download/vo.py

```diff
@@ -59,13 +60,49 @@ def run(self, dirpath: str, local_authorities: List[str]):
             dirpath (str): Path to directory where data will be saved
             local_authorities (List[str]): Names of local authorities to be queried
         """
+
         savedir = path.join(dirpath, "vo")
 
         if path.exists(savedir):
-            self.logger.info(f"Skipping as {savedir} exists!")
+            logger.info(f"{savedir} already exists")
+
         else:
+
             mkdir(savedir)
-            _download_by_local_authority(savedir, local_authorities)
+
+            categories = [
+                "OFFICE",
+                "FUEL/DEPOT",
+                "LEISURE",
+                "INDUSTRIAL USES",
+                "HEALTH",
+                "HOSPITALITY",
+                "MINERALS",
+                "MISCELLANEOUS",
+                "RETAIL (SHOPS)",
+                "UTILITY",
+                "RETAIL (WAREHOUSE)",
+                "NO CATEGORY SELECTED",
+                "CENTRAL VALUATION LIST",
+                "CHECK CATEGORY",
+                "NON-LIST",
+                "NON-LIST EXEMPT",
+            ]
+
+            for local_authority in local_authorities:
+                for category in categories:
+                    category_without_slashes = category.replace("/", " or ")
+                    try:
+                        download(
+                            url=f"https://api.valoff.ie/api/Property/GetProperties?Fields=*&LocalAuthority={local_authority}&CategorySelected={category}&Format=csv&Download=true",
+                            filepath=path.join(
+                                savedir,
+                                f"{local_authority} - {category_without_slashes}.csv",
+                            ),
+                        )
+
+                    except requests.HTTPError as error:
+                        logger.info(error)
```

Comment:
```suggestion
_download_by_local_authority(savedir, local_authorities)
```
Has been refactored into a function in prior PR